### ISSUE TITLE
Ask for superuser password only once; Open Polkit UI directly

### ISSUE THREAD

**Issue by KSXGitHub** [microsoft/vscode#161451]
I like using VS Code to edit my Linux system configuration file. One inconvenience that annoys me is that VS Code would ask for superuser password every time I press <kbd>Ctrl+S</kbd>. Not to mention, I must switch to my mouse to access the error box to click ["Retry as Sudo"](https://user-images.githubusercontent.com/11488886/191644299-9cbd8f94-0ab1-4049-a9fb-94069fa78d13.png). which significantly slows me down.

**Suggestion:**
1. Only ask for superuser password once. VS Code may ask for superuser password again if Polkit credential is expired.
2. Do not show [this error notification](https://user-images.githubusercontent.com/11488886/191644464-d7ba34b4-d74e-403c-a7bf-0066ee4181ba.png). Use Polkit to ask user for password directly instead. ~~Alternatively, make the error notification reachable by keyboards alone (no mouse should be involved).~~

**Comment by yume-chan** [comment]
> make the error notification reachable by keyboards alone

F1 > Notifications: Focus Notification Toast, or you can add a custom keybinding for it.

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by scarf005** [comment]
> F1 > Notifications: Focus Notification Toast

however, it still does not solve that for every save, passwords needs to be typed.
it is a significant disadvantage, for example `sudo nano` or `sudo vim` does not suffer from this problem.
since vscode discourages to be launched as root, it'd be a good idea to use polkit as OC stated.

**Comment by VSCodeTriageBot** [comment]
This feature request has not yet received the 20 community [upvotes](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) it takes to make to our backlog. 10 days to go. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_smiling_face: This feature request received a sufficient number of community upvotes and we moved it to our backlog. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!
